src/dotnet/Chat.Service/ChatsUpgradeBackend.InitializeData.cs

```diff
@@ -358,4 +358,124 @@ string ConvertOldTextToTimeMap(string textToTimeMapJson)
             return textToTimeMapJson;
         }
     }
+
+    // [CommandHandler]
+    public virtual async Task<Chat> CreateFeedbackTemplateChat(IChatsUpgradeBackend.CreateFeedbackTemplateChatCommand command, CancellationToken cancellationToken)
+    {
+        if (Computed.IsInvalidating())
+            return default!; // It just spawns other commands, so nothing to do here
+
+        var chatId = Constants.Chat.FeedbackTemplateChatId;
+        var usersTempBackend = Services.GetRequiredService<IUsersUpgradeBackend>();
+        var hostInfo = Services.GetRequiredService<HostInfo>();
+        var userIds = await usersTempBackend.ListAllUserIds(cancellationToken).ConfigureAwait(false);
+
+        var admin = await AccountsBackend.Get(Constants.User.Admin.UserId, cancellationToken)
+            .Require()
+            .ConfigureAwait(false);
+        var creatorId = UserId.None;
+
+        var userIdByEmail = new Dictionary<string, UserId>(StringComparer.OrdinalIgnoreCase);
+        foreach (var userId in userIds) {
+            var account = await AccountsBackend.Get(userId, cancellationToken).ConfigureAwait(false);
+            if (account == null)
+                continue;
+
+            var user = account.User;
+            if (user.Claims.Count == 0)
+                continue;
+            if (!user.Claims.TryGetValue(ClaimTypes.Email, out var email))
+                continue;
+
+            if (hostInfo.IsDevelopmentInstance) {
+                if (email.OrdinalIgnoreCaseEndsWith("actual.chat"))
+                    userIdByEmail.Add(email, userId);
+            }
+            else {
+                if (OrdinalIgnoreCaseEquals(email, "alex.yakunin@actual.chat")
```

Comment:
May be better to extract your emails in constants. There is already another place with them. See ChatsUpgradeBackend.CreateAnnouncementsChat